config/core/base.pan

```diff
@@ -55,14 +55,38 @@ variable YUM_OS_DISTRIBUTION_NAME ?= {
   };
 };
 
+@{
+desc = use iptables and ip6tables services instead of firewalld
+value = true or false
+default = false (EL7 default is to use firewalld)
+required = no
+}
+variable OS_USE_IPTABLES_SERVICES ?= false;
+
+
+@{
+desc = disable firewalld service
+value = true, false or undef/null
+default = true if OS_USE_IPTABLES_SERVICES=true else undef (nothing done)
```

Comment:
This is a comment! Seems clear to me what the meaning is! You are not supported to copy/execute this part of the sentence (which is far from being a valid pan statement).